partcad/src/partcad/part_factory_3mf.py

```
@@ -9,10 +9,12 @@
 
 import build123d as b3d
 
+from .sentry import instrument
 from .part_factory_file import PartFactoryFile
 from . import logging as pc_logging
 
 
+@instrument()
 class PartFactory3mf(PartFactoryFile):
     def __init__(self, ctx, source_project, target_project, config):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)